haproxy_config/haproxy_config.tmpl

```diff
@@ -93,6 +101,13 @@ frontend https-in
 backend {{container.name}}_cluster
     mode http
     server node1 {{container.ip}}:{{container.port}}
+    {% if container.basic_auth %}
+    acl draw-auth http_auth({{container.name}}_list)
+    http-request auth realm draw unless draw-auth
+    {% endif %}
+    # Strange behaivor ...
+    # Remove authentication header from backend
+    http-request del-header Authorization
```

Comment:
shouldnt this be also encapsulated by the if condition? Otherwise it might remove authorization headers which might be handled by the container (like shield or any APIs)

Reply:
hi @mbutkereit

this is the last open question from my side. What do you think?

Reply:
In the first attempt I had problems with it, now in the second it looked good. So I put it back into the if statement.